package.json

```
@@ -64,9 +75,8 @@
     "url": "https://github.com/digitalbazaar/http-client/issues"
   },
   "homepage": "https://github.com/digitalbazaar/http-client",
-  "module": "main.js",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.0.0"
```

Due to `ky` updates need to be on at least 14.  Not sure which minor, guessing here.
```suggestion
    "node": ">=14.14.0"
```